Fix :::details and add Shield explanation (#478)
zkSNACKs/WasabiDoc: docs/glossary/Glossary-PrivacyWasabi.md

They are literally peers in the network, or in the CoinJoin.
:::

:::details

> ### Shield (or anonset shield/anonimity set shield)

Reviewer: ```suggestion
### Shield (or anonset shield/anonymity set shield)
```

We often talk about green, yellow and red shields... but what are they?

> Coins in your wallet have shields (red, yellow, green and green +).

Reviewer: ```suggestion
Coins in your wallet have shields (red, yellow, green and green check-mark).
```

> Each shield represents the [Anonimity Set](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set) of a given coin.

Reviewer: ```suggestion
Each shield represents the [Anonymity Set](/FAQ/FAQ-UseWasabi.md#what-is-the-anonymity-set) level of a given coin.
By the anonymity set levels are `2`, `21` and `50`, however, this can be [changed in the settings](/FAQ/FAQ-UseWasabi.md#how-can-i-change-the-anonset-target).
```

> Essentially, when we talk about shields, we indicate a specific Privacy Level.

Reviewer: ```suggestion
Essentially, when we talk about shields, we mean a specific privacy level set by the user.
```

They are literally peers in the network, or in the CoinJoin.
:::

:::details

> ### Shield (or anonset shield/anonymity set shield)

Reviewer 2: ```suggestion
### Shield (or anonset shield)
```

> We often talk about green, yellow and red shields... but what are they?

Reviewer 2: ```suggestion
We often talk about check mark, green, yellow and red shields... but what are they?
```
